RecoHGCal/TICL/plugins/PatternRecognitionbyCLUE3D.cc

```diff
@@ -718,7 +719,7 @@ void PatternRecognitionbyCLUE3D<TILES>::fillPSetDescription(edm::ParameterSetDes
   iDesc.add<bool>("densityOnSameLayer", false);
   iDesc.add<double>("criticalEtaPhiDistance", 0.035);
   iDesc.add<double>("outlierMultiplier", 2);
-  iDesc.add<int>("minNumLayerCluster", 5)->setComment("Not Inclusive");
+  iDesc.add<int>("minNumLayerCluster", 2)->setComment("Not Inclusive");
```

Comment:
(As above, I have surely missed something). Why is this going  from `5` to `2`?

Reply:
Ciao @AdrianoDee in this case there's no _real_ reason. The idea, in this case, is that the selection will be done *on top* of this and will likely use more advanced ML techniques. For this reason, it could be better to allow more input to possibly gather a little more in output. I hope this is clear.

Reply:
I see. Thanks @rovere for all the details (here and above).